### DISCUSSION TITLE
能不能弄个是否保留日志文件的选项，整天给我电脑弄一堆日志...

### DISCUSSION THREAD

**Discussion by gdxdf2** [Meloong-Git/PCL#1742] [提问] [answered]
### **能不能弄个是否保留日志文件的选项，整天给我电脑弄一堆日志...**

**Comment by Pigeon0v0** [comment]
展开讲讲是什么日志

**Comment by Chiloven945** [comment, marked as answer]
PCL的log1~5是为了在遇到bug的时候好进行分析
游戏的log你可以使用百宝箱内的清理垃圾进行清理

**Comment by MoYuan-CN** [comment]
PCL2 本体目录下 PCL 文件夹中的 Log1~5 是启动器日志，可方便分析Bug（这玩意也不会占多大硬盘空间的）
游戏日志应当只会在游戏崩溃时保存，位置在客户端对应目录中的 `~\crash-report` 目录下，可手动清除
缓存的文件可以使用百宝箱内的清理垃圾进行清理

当然，如果以上内容无法满足您的需求，可到 [这里](https://github.com/Hex-Dragon/PCL2/issues/new/choose) 提交一个 优化/新功能提案 反馈

**Reply by Chiloven945** [reply]
游戏日志同样会被保存在logs文件夹下
![image](https://user-images.githubusercontent.com/116699482/235337794-84f9df42-e700-46b2-b1b2-8444c2d65818.png)

**Reply by MoYuan-CN** [reply]
啊这。。。那就麻烦了
交反馈吧

**Comment by gdxdf2** [comment]
后缀是.log的文件
        
        
            
        
    
         
        
            
                        ***@***.***
            
        
     
    
  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2023年4月30日 11:14
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [Hex-Dragon/PCL2] 能不能弄个是否保留日志文件的选项，整天给我电脑弄一堆日志... (Discussion #1742)
        
      
    
  

展开讲讲是什么日志
